### ISSUE TITLE
Module ISRM-S-X10S Antenna selection options internal/external on model level effect global change

### ISSUE THREAD

**Issue by elekSENDER** [EdgeTX/edgetx#3403]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

In a X10S with an ISRM upgrade modul at Model Setup/Internal RF/Module Options there is a possibilty in each model to change ext/int antenna. By changing this parameter from int to ext and viceversa this will effect not only the selected model but all models.

### Expected Behavior

The setted value should remain set and used for each model.
The other possibility is to move this kind of setting to a system setting page and remove it from model level.

### Steps To Reproduce

At model A select ISRM internal antenna.
Change at model B from internal to external antenna.
Check at model A antenna setting: it changed to external.

### Version

Other (Please specify below)

### Transmitter

FrSky X10 / X10S (ACCST)

### Anything else?

Firmware: x10-2.8.1-d9d4a8a
Transmitter: Frsky Horus X10s + ISRM upgrade module

**Comment by pfeerick** [comment]
Just to be clear - does this mean you select Internal RF -> XJT or ISRM? As this changes how this option is managed.

**Comment by elekSENDER** [comment]
Internal RF is ISRM.
And sorry, the above description is not completely correct: I can´t switch between int./ext. antenna, I mean the window where I can switch Ext. antenna on/off.

**Comment by pfeerick** [comment]
I think the setting you are changing is a module-level setting, not a model or system setting. It also may in fact not change every model, but every model with the same receiver number. If this is a module-level setting, I don't think there is anything we can do about this. 

i.e. Is this an option shown when you go into here? (I can't see that in simulator since everything shown on that screen is determined by the module,  and I haven't used any ISRM/ACCESS stuff with dual antenna options.

![image](https://user-images.githubusercontent.com/5500713/227747729-b3491170-2bfd-4855-b186-ce1832cc77f3.png)

**Comment by elekSENDER** [comment]
> but every model with the same receiver number.

This occurs at 2 different models with 2 different RX with 2 different RX numbers.

Let me try to explain via screenshots direct from the transmitter:
Model A:
![grafik](https://user-images.githubusercontent.com/98912034/227763752-d6302222-4f33-43e8-8560-4fa9dd1a9f82.png)

select button you signed above with red arrow:
![grafik](https://user-images.githubusercontent.com/98912034/227763805-18b20189-8fb4-4f41-a6bf-e4a574e39bb3.png)
ext. antenna is ON.

change to model B, different RX number and RX:
![grafik](https://user-images.githubusercontent.com/98912034/227763888-3bce1272-2427-4db3-be62-d1ddc546e1c7.png)

enter Module options:
![grafik](https://user-images.githubusercontent.com/98912034/227765599-cdcfd536-e996-469a-9bd7-99bbaf2985df.png)

switch Ext. antenna to OFF and save
![grafik](https://user-images.githubusercontent.com/98912034/227763955-7a0703d5-a22c-4931-b579-149bcf4c1ff2.png)

Change back to model A:
![grafik](https://user-images.githubusercontent.com/98912034/227764067-0fde4179-269f-400f-89d2-4e4474b4ae08.png)

Verify antenna settings:
![grafik](https://user-images.githubusercontent.com/98912034/227764104-9752c75d-b48e-4097-9a0b-f1609383ad83.png)
previously exited with ON, now OFF.

**Comment by elekSENDER** [comment]
I understand the changing on module level, but if there is no way to initalize the module differently with each model then it would be better to make this setting maybe anywhere around the module selection in the hardware section:
![grafik](https://user-images.githubusercontent.com/98912034/227766270-bb0c6ce0-c428-4830-bd28-09de7b2b7ee4.png)

Just to prevent that you think you make the antenna selection by changing a model and screw off the external antenna without really changing the module setting.

**Comment by pfeerick** [comment]
Yes, indeed - I'll find out if there is anything there that would be model level, and if not, we can then update this to be a "move the module settings to the hardware page" request.
